docs/examples.rst

```diff
@@ -58,6 +60,24 @@ arbitrary sequence of OpenGL commands into a NanoGUI widget.
 - `Example 4 in C++ <https://github.com/wjakob/nanogui/blob/master/src/example4.cpp>`_
 - `Example 4 in Python <https://github.com/wjakob/nanogui/blob/master/python/example4.py>`_
 
+.. _nanogui_example_icons:
+
+Example Icons
```

Comment:
These were added for testing, but I don't see the harm in having them.  The icons will show up slightly differently than the webpage because of the default dark background of NanoGUI.  Can remove if undesired.

Reply:
Neat. Would you mind calling the executable ``example_icons``?

Reply:
Done